### DISCUSSION TITLE
[Feature Request] Searching by rating/field like in Clementine

### DISCUSSION THREAD

**Discussion by Asmageddon** [strawberrymusicplayer/strawberry#946] [Ideas]
As an example, in Celementine you could search the current playlist by e.g. `rating:>3`, which would find songs rated 3.5+ in Clementine. 

No such behavior exists in Strawberry, and I would like to have it.

**Comment by Asmageddon** [comment]
@jonaski Actually, fiddling with this, it seems that this exists in Strawberry, it's just broken. Searching for e.g. `rating:3`, or other numbers, sometimes filters out songs from the list, but there's no apparent sense to how it does it.

**Comment by happy-turtle1** [comment]
Does anyone know where I can download Strawberry 0.8.4 for mac?  I think that is the best version for my old Mac OS 10.14.6 Mojave.  Any help would be greatly appreciated.

**Comment by Asmageddon** [comment]
Bump since this is the #1 feature I miss in Strawberry compared to Clementine.
